### PR TITLE
Added 16:9 and square images, updated quality

### PR DESCRIPTION
**Pull request by peterp**

### CODE REVIEW

File: gem/static/css/enhanced.scss [hunk 1]
```diff
@@ -47,7 +47,7 @@
 body {
   font-size: 18px;
   line-height: 24px;
-  max-width: 320px;
+  max-width: 400px;
 }
```
Comment (saeedmb8) [inline gem/static/css/enhanced.scss]: @peterp Is there any specific reason that we are changing this to `400px`?

Comment (peterp) [inline gem/static/css/enhanced.scss]: @saeedmb8 for phones in landscape; ideally this should actually just resize to 100%, and take up all the available space on the device. But we're busy showing the site to people and they get confused when it takes up all the space.

Comment (peterp) [inline gem/static/css/enhanced.scss]: @saeedmb8 what do you guys typically do?

**Comment by saeedmb8** [comment]
:+1:

**Pull request merged**